system/system.proto

```diff
@@ -124,7 +124,7 @@ enum RebootMethod {
   HALT = 3;        // Halt, if possible.
   WARM = 4;        // Reload configuration but not underlying hardware.
   NSF = 5;         // Non-stop-forwarding reboot, if possible.
-  RESET = 6;       // Reboot to factory defaults.
+  reserved 6;      // Please use instead gNOI FactoryReset.Initiate().
```

Comment:
maybe something like `RESET method is deprecated in favor of the gNOI FactoryReset.Initiate()`

Reply:
Thanks!